src/main/java/io/vertx/ext/httpservicefactory/package-info.java

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
URLs are free form (i.e http/https), there is no special format like in Vert.x 2.